composer.json

```diff
@@ -1,20 +1,20 @@
 {
     "name": "friendsofcake/cakephp-csvview",
-    "description": "A CSV View class for CakePHP 3.x",
+    "description": "A CSV View class for CakePHP",
     "type": "cakephp-plugin",
     "keywords": [
         "cakephp",
         "csv",
         "view",
         "export"
     ],
-    "homepage": "http://github.com/friendsofcake/cakephp-csvview",
+    "homepage": "https://github.com/friendsofcake/cakephp-csvview",
     "license": "MIT",
     "authors": [
         {
             "name": "Jose Diaz-Gonzalez",
             "email": "email@josediazgonzalez.com",
-            "homepage": "http://josediazgonzalez.com",
+            "homepage": "https://josediazgonzalez.com",
```

Comment:
This site doesn't work on HTTPS :P

Reply:
Geeze

Reply:
It's on github pages and the ssl cert hasn't finished generating? Not sure what thats about...

Reply:
https://josediazgonzalez.com  is accessible now.